test.js

```diff
@@ -1,34 +1,35 @@
-var tape = require('tape')
-var pager = require('./')
+const tape = require('tape')
+const Pager = require('./')
+const crypto = require('crypto')
 
 tape('get page', function (t) {
-  var pages = pager(1024)
+  const pages = new Pager(1024)
 
-  var page = pages.get(0)
+  const page = pages.get(0)
 
   t.same(page.offset, 0)
   t.same(page.buffer, Buffer.alloc(1024))
   t.end()
 })
 
 tape('get page twice', function (t) {
```

Comment:
A potential improvement could be to replace magic values (0, 1024) by variables like you did in the [cases you add](https://github.com/mafintosh/memory-pager/pull/4/files#diff-13876b4beb64b9f156474dc78f9c923952a7ca210d4507b6b3135bbe244f8a60R83).

Let me know if I can help ^^

Reply:
Indeed, these could/should be constants (`pageSize`) I chose 1024 because it seemed like a standard value, but I guess any size would work. Smaller = faster?!

_Note: I invited you to my fork, feel free to update the PR if you like_

Reply:
🙌  I'll try to find a slot to add it ^^ But we could add it in another PR anyway.

Reply:
Certainly. I think this should be okay to review @mafintosh